lib/Chat/Parser/Command.php

```diff
@@ -48,6 +48,7 @@ public function parseMessage(Message $message): void {
 			return;
 		}
 
+		$message->setMessageType('command');
```

Comment:
[The message type is set based on the verb](https://github.com/nextcloud/spreed/blob/6fb78781cdd925946360fed0f8fc2776f1617a19/lib/Chat/MessageParser.php#L69) and [the verb for commands is set to `command` when they are executed](https://github.com/nextcloud/spreed/blob/master/lib/Chat/Command/Executor.php#L101). Therefore, could that be removed or is it needed in some case?

Reply:
right, I added this without checking.

I guess we need to expose it to the API as well, so the mobile apps can check if replying is allowed as well.

Reply:
> I guess we need to expose it to the API as well, so the mobile apps can check if replying is allowed as well.

Makes sense :+1: